Automattic/sensei
# Add/course progress block colors

### Changes proposed in this Pull Request

* Adds coloring controls for the progress bar.
* I ended up adding a control for both the bar background and the bar because if we do not set a value for the background it appears differently in different browsers.
* The coloring of the progress element is controlled by pseudo elements making it a bit problematic to change it in PHP/JS. For this reason I ended up using CSS variables. It seems that it is not supported by IE 11, not sure if this is an issue.
* For the same reason I couldn't use the `withColorSettings` HOC.
* Any recommendations on the format on the bar and the default colors welcome.

### Testing instructions

* Open a course and add a 'Course progress bar' block.
* Save it and verify that it looks ok with the default colors.
* Try different colors and observe the changes both in the editor and in backend.
